server/shared/publishing/helpers.js

```diff
@@ -47,6 +47,7 @@ function publishActivity(activity) {
 
     activity.publishedAt = new Date();
     addToSpine(spine, activity);
+    Object.assign(spine, getRepositoryAttrs(repository));
```

Comment:
Would be good idea to create new method to handle this e.g. `applyAttributesToSpine` and apply same method to `publishRepositoryDetails` since, any changes to publishing repository details should be applied when publishing activity.